### ISSUE TITLE
Package has orphaned 3.0.0 version

### ISSUE THREAD

**Issue by dave-irvine** [JamieDixon/react-lifecycle-component#15]
Hi!

What's up with this 3.0.0 release? I seem to have installed it somehow... https://www.npmjs.com/package/react-lifecycle-component/v/3.0.0

I can't find any reference to it in the repo history, so just a bit concerned where it has come from.

**Comment by JamieDixon** [comment]
@dave-irvine Hmmmm 🤔 There's 19 versions of the library published to npm and the latest one is 2.0.6. I can't see any reference to 3.0.0 although your link works fine. I'm a bit baffled right now. I'll look into it and see if anything emerges.

Thanks for the heads-up 👍

**Comment by dave-irvine** [comment]
It's not the end of the world assuming you don't find anything dodgy, but you will need to publish a new `4.0.0` release as people other than me will now be "stuck" on the orphaned major version, so no new `2.x` releases will show up for them when checking for updates.

**Comment by JamieDixon** [comment]
I'm going to see if I can get to the bottom of this 3.0.0 business, and if not I'll just push a 4.0.0. I'll report back here when I find something or re-publish.

**Comment by JamieDixon** [comment]
@dave-irvine After investigating this issue, it seems this is a user error on my part.

Over a year ago I published version 3.0.0 without making any new commits to the library. I've `diff`ed the 3.0.0 version with the recent 2.0.6 version and the only differences were recent commits.

To that end I've taken the most recent commit line, 2.0.6, and re-published it under the 4.0.0 version number.

A new release has been drafted here: https://github.com/JamieDixon/react-lifecycle-component/releases/tag/4.0.0 and the library has been published to npm https://www.npmjs.com/package/react-lifecycle-component

My apologies for this mistaken. I'm unsure exactly why I published the 3.0.0 version but it may have related to a project I was working on at the time.
